server/apps/postgres-app/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+# In order to start James JPA app on top of mariaDB:
+# 1. Download the driver: `wget https://repo1.maven.org/maven2/org/mariadb/jdbc/mariadb-java-client/2.7.2/mariadb-java-client-2.7.2.jar`
+# 2. Generate the keystore with the default password `james72laBalle`: `keytool -genkey -alias james -keyalg RSA -keystore keystore`
+# 3. Start MariaDB: `docker-compose up -d mariadb`
+# 4. Start James: `docker-compose up james`
+
+services:
+
+  james:
+    depends_on:
+      - mariadb
+    image: apache/james:postgres-latest
+    container_name: james
+    hostname: james.local
+    volumes:
+      - /home/tungtv/workplace/JAMES/tmail-backend/james-project/server/apps/postgres-app/mariadb-java-client-2.7.2.jar:/root/libs/mariadb-java-client-2.7.2.jar
+      - /home/tungtv/workplace/JAMES/tmail-backend/james-project/server/apps/postgres-app/sample-configuration/james-database-mariadb.properties:/root/conf/james-database.properties
+      - /home/tungtv/workplace/JAMES/tmail-backend/james-project/server/apps/postgres-app/src/test/resources/keystore:/root/conf/keystore
+
+  mariadb:
+    image: mariadb:10.6
```

Comment:
Can we point JPA to Postgres for now?

Reply:
It is not related, I don't think it necessary.

Reply:
Disagree... 

```
DOD: apache/james:pg-latest docker image + docker-compose
```
I would expect postgresql instead of mariadb in the docker-compose :)